helpers/migrate4.py

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/env python3
```

Comment:
This script is based on the [GKE for_each migrate script ](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/helpers/migrate7.py). The only changes is `MIGRATIONS` list + copyright year bump